nbconvert/templates/latex/document_contents.tplx

```diff
@@ -30,12 +30,12 @@
 
 % Display latex
 ((* block data_latex -*))
-    ((( output.data['text/latex'] | citation2latex | strip_files_prefix | markdown2latex )))
+    ((( output.data['text/latex'] | citation2latex | strip_files_prefix | convert_pandoc('markdown', 'latex'))))
```

Comment:
@Carreau (see earlier comment) in this case it's being invoked without a source argument at all, and no named arguments for the format